site-src/implementations.md

```diff
@@ -5,8 +5,100 @@ and provides status and resource references for them.
 
 Implementors and integrators of Gateway API are encouraged to update this
 document with status information about their implementations, the versions they
-cover, and documentation to help users get started.
-
+cover, and documentation to help users get started. This status information should
+be no longer than a few paragraphs. 
+
+There are three main classes of implementation:
+
+* **Conformant** implementations - these implementations have submitted at least
+  one conformance report passing all claimed features for one of the two (2) most
+  recent Gateway API releases.
```

Comment:
```suggestion
  one conformance report passing all tests for ["Core" support level](https://gateway-api.sigs.k8s.io/concepts/conformance/#2-support-levels)
  features and all claimed "Extended" features for one of the two (2) most recent
  Gateway API releases.
```
Trying to be a touch more specific that all Core support level features are required and skipping tests within a claimed feature disqualifies a "Conformant" status. Should the support levels link be site-root relative instead?

Reply:
Yea that makes sense to me @robscott

Reply:
Updated, thanks.